SPECS/ocaml-mirage-block-volume.spec

```diff
@@ -0,0 +1,79 @@
+%define debug_package %{nil}
+
+Name:           ocaml-mirage-block-volume
+Version:        0.9.0
+Release:        1%{?dist}
+Summary:        Mirage volume library compatible with LVM
+License:        ISC
+URL:            https://github.com/mirage/mirage-block-volume/
+Source0:        https://github.com/mirage/mirage-block-volume/archive/v%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  ocaml
+BuildRequires:  ocaml-ocamldoc
+BuildRequires:  ocaml-camlp4-devel
+BuildRequires:  ocaml-findlib
```

Comment:
BTW released versions shouldn't use bisect because

- it'll fill up your disk with bisect output files
- it'll slow everything down
- it's GPLv3 and most of these libraries are supposed to be ISC

I've been removing the bisect build depend when releasing the libraries.


On Mon, Apr 13, 2015 at 10:30 AM, Jon Ludlam <notifications@github.com>
wrote:

> In SPECS/ocaml-mirage-block-volume.spec
> <https://github.com/xenserver/buildroot/pull/635#discussion_r28224932>:
>
> > @@ -0,0 +1,79 @@
> > +%define debug_package %{nil}
> > +
> > +Name:           ocaml-mirage-block-volume
> > +Version:        0.9.0
> > +Release:        1%{?dist}
> > +Summary:        Mirage volume library compatible with LVM
> > +License:        ISC
> > +URL:            https://github.com/mirage/mirage-block-volume/
> > +Source0:        https://github.com/mirage/mirage-block-volume/archive/v%{version}/%{name}-%{version}.tar.gz
> > +BuildRequires:  ocaml
> > +BuildRequires:  ocaml-ocamldoc
> > +BuildRequires:  ocaml-camlp4-devel
> > +BuildRequires:  ocaml-findlib
>
> Right, I should fix this and the spec I copied it from as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xenserver/buildroot/pull/635/files#r28224932>.
>



-- 
Dave Scott